swagger.yaml

```diff
@@ -321,4 +329,7 @@ definitions:
   Limit:
     type: string
     description: "The limit applied to the number of results returned"
```

Comment:
Limit is actually more like the following:

"The number of items requested, defaulted to 50 and limited to 1000." <- from search api